cvat/apps/lambda_manager/views.py

```diff
@@ -0,0 +1,341 @@
+from django.shortcuts import render
+from rest_framework import viewsets, status
+from rest_framework.response import Response
+from cvat.apps.engine.frame_provider import FrameProvider
+from cvat.apps.engine.models import Task as TaskModel
+from cvat.apps.dataset_manager.task import (put_task_data, patch_task_data,
+    delete_task_data)
+from django.core.exceptions import ValidationError
+import base64
+import json
+import requests
+import django_rq
+import rq
+from functools import wraps
+from cvat.apps.engine.serializers import LabeledDataSerializer
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+
+class LambdaGateway:
+    NUCLIO_ROOT_URL = '/api/functions'
+
+    def _http(self, method="get", scheme=None, host=None, port=None,
+        url=None, data=None):
+        NUCLIO_GATEWAY = '{}://{}:{}'.format(
+            scheme or settings.NUCLIO['SCHEME'],
+            host or settings.NUCLIO['HOST'],
+            port or settings.NUCLIO['PORT'])
+        NUCLIO_HEADERS = {'x-nuclio-project-name': 'cvat'}
+        NUCLIO_TIMEOUT = settings.NUCLIO['DEFAULT_TIMEOUT']
+
+        if url:
+            url = "{}{}".format(NUCLIO_GATEWAY, url)
+        else:
+            url = NUCLIO_GATEWAY
+
+        reply = getattr(requests, method)(url, headers=NUCLIO_HEADERS,
+            timeout=NUCLIO_TIMEOUT, json=data)
+        reply.raise_for_status()
+        response = reply.json()
+
+        return response
+
+    def list(self):
+        data = self._http(url=self.NUCLIO_ROOT_URL)
+        response = [LambdaFunction(self, item) for item in data.values()]
+        return response
+
+    def get(self, id):
```

Comment:
Codacy found an issue: [Redefining built-in 'id'](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)